ush/config.sh.RRFS_NA_13km

```diff
@@ -21,6 +21,7 @@ RUN_ENVIR="nco"
 PREEXISTING_DIR_METHOD="rename"
 
 PREDEF_GRID_NAME=GSD_RAP13km
+ADDNL_OUTPUT_GRIDS=( "130" "242" )
```

Comment:
I've had some issues recently with using arrays in the configs. How does this one show up in `var_defns.sh`? As expected?

Reply:
Yeah this took some trial and error.  It shows up correctly as an array in var_defns.sh, although the formatter puts each array item on a new line.  By pointing to some real-time North America dev1 output grids, I confirmed that exregional_run_post.sh reads in this array correctly and produces the desired output, whether there are 1 or more output grids listed, or if the array is empty like the default.

Reply:
Perfect. Thanks for checking in on that. The arrays are tough!